api/filters/replacement/replacement.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package replacement
+
+import (
+	"fmt"
+	"strings"
+
+	"sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+type Filter struct {
+	Replacements []types.Replacement
+}
+
+const DefaultFieldPath = ".metadata.name"
```

Comment:
Please add a comment as to why the leading period is needed.

Reply:
It isn't needed and I took it out